### ISSUE TITLE
Consider equalEarth as default projection type

### ISSUE THREAD

**Issue by jwoLondon** [vega/vega#1985]
Now that equalEarth is one of the core Vega projection types, would you consider making it the default projection type rather than mercator? Apart from its familiarity, I see little advantage in using the mercator projection for most cartographic data. It is particularly problematic for global maps that don't focus on high latitudes.

Equal Earth with its proportional area projection, horizontal lines of latitude, no extreme shape distortion and curved bounds only for oceanic areas provides a much more suitable default for the kinds of mapping to which Vega is suited.

I think there is also an argument for making the identity projection the default given this Is conceptually close to 'unprojected'. Although on balance this may cause backward compatibility problems without reflecting the y scale.

See also https://github.com/vega/vega-lite/issues/5367

**Comment by jheer** [comment]
Thanks for the suggestion! I agree with most of the reasoning, but am concerned about backwards compatibility. This change could break a significant number of existing specifications that assume the default Mercator projection. So I'm not sure if the benefits outweigh the cost. In any case, as this is arguably a breaking change, it would necessitate a major version bump.

If folks have thoughts on the matter, they should feel free to chime in!

**Comment by jwoLondon** [comment]
Fair point. Perhaps reserve for the next major version change with some documentation pointers in the meantime?

Despite attempting to teach students to be sceptical about map projection choice, I see way too much uncritical use of the mercator projection, so in the long term, I think a mercator 'opt-in' rather than 'opt-out' would benefit the quality of geovis in the Vega universe.

**Comment by jheer** [comment]
As a compromise measure, I've added configuration support for projections, to be released in v5.7.0. This will allow changing the default projection in one's own deployments.

**Comment by domoritz** [comment]
Vega-Lite is changing the default projection to equalEarth in Vega-Lite 5. We can remove the explicit default in Vega's next major version.

**Comment by hydrosquall** [comment]
Another option the avoids any backward compat issues would be to make this the primary example in the examples gallery for maps. 

Many users are copying from the gallery for their first graphs anyways, rather than starting from an empty json.

**Comment by domoritz** [comment]
I can go either way. Using a better projection in the examples is a good step.
